egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -86,6 +99,16 @@ def GetArgs():
     parser.add_argument("--lstm-delay", type=str, default=None,
                         help="option to have different delays in recurrence for each lstm")
 
+    # Options to convert input MFCC into Fbank features. This is useful when a
+    # LDA layer is not added (such as when using dense targets)
+    parser.add_argument("--cepstral-lifter", type=float, dest = "cepstral_lifter",
```

Comment:
Is there a reason why you need to use filterbanks instead of MFCCs?  E.g. you have a convolutional architecture?

The notion that may people have Fbank being better than MFCCs is mostly a misunderstanding, since filterbanks are generally extracted with a higher dimension and that's what matters.  If this turns out to be removable, it would be a good simplification.